lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -321,8 +322,8 @@ function dosomething_campaign_preprocess_hot_module(&$vars, &$wrapper) {
     array('dosomethingCampaign' =>
       array(
         'goals' => array(
-          'dates' => json_encode($dates),
-          'quantities' => json_encode($quantities),
+          'dates' => (isset($dates)) ? json_encode($dates) : null,
```

Comment:
@DFurnes my bad. I didn't see this comment! Yea, I don't think it is necessary. Going to remove it and rework the client side stuff. Thanks!